lms/static/scripts/frontend_apps/components/GradingControls.tsx

```diff
@@ -10,6 +10,7 @@ import SubmitGradeForm from './SubmitGradeForm';
 
 export type GradingControlsProps = {
   students: StudentInfo[];
+  scoreMaximum?: number | null;
```

Comment:
I made the prop optional here because this component is not using it, just passing it down to `SubmitGradeForm`. That way we don't need to change the test for this component.

It feels a bit like prop drilling. I considered making `SubmitGradeForm` fetch the value using `useConfig`, but on the other hand it feels like keeping `InstructorToolbar` as the only component consuming `useConfig`, is the right approach.